tests/cheriot/builtins.cpp

```diff
@@ -0,0 +1,181 @@
+#include "../rego_test.h"
```

Comment:
Has this code come from cheriot?  Do we need some attribution here?

Reply:
This is our SPDX header:

```
// Copyright SCI Semiconductor and CHERIoT Contributors.
// SPDX-License-Identifier: MIT
```